test/app_signals/traces_test.go

```diff
@@ -0,0 +1,110 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+//go:build !windows
+
+package app_signals
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/xray"
+
+	"github.com/aws/amazon-cloudwatch-agent-test/test/status"
+	"github.com/aws/amazon-cloudwatch-agent-test/test/test_runner"
+	"github.com/aws/amazon-cloudwatch-agent-test/util/awsservice"
+)
+
+const (
+	lookbackDuration     = time.Duration(-80) * time.Minute
```

Comment:
80 minutes seems like a long time. How long is this test supposed to run?

Reply:
Changed it to 30 assuming the metrics are generated as part of validation itself, and each validation is retried every 30 sec for 6 times.